spartan/aztec-network/eth-devnet/config/val-keys/data/keys/0x82626f97b0fc009c2d2e098e2c3b21922c6525d314552674899953fbb1c4cb48d01966e80167f81b1a36e78e2a6362a9/voting-keystore.json

```diff
@@ -0,0 +1 @@
+{"crypto":{"kdf":{"function":"pbkdf2","params":{"dklen":32,"c":262144,"prf":"hmac-sha256","salt":"4fe22290246afebf4d2b064581ba0cec5853b0e8a1de24b5262e0c82fc265f8e"},"message":""},"checksum":{"function":"sha256","params":{},"message":"3449e5f82bd7470632dbc5d672c09dbe07b0d56e3afda32583c3a57000aebde4"},"cipher":{"function":"aes-128-ctr","params":{"iv":"fa773de4e25f34a4ffa61fcde5bbaa3b"},"message":"2422da8705b02af0124aebcbbc0ef713f051a96f3b0865ea830dd9817e0ff88d"}},"description":"0x82626f97b0fc009c2d2e098e2c3b21922c6525d314552674899953fbb1c4cb48d01966e80167f81b1a36e78e2a6362a9","pubkey":"82626f97b0fc009c2d2e098e2c3b21922c6525d314552674899953fbb1c4cb48d01966e80167f81b1a36e78e2a6362a9","path":"","uuid":"2ddc2811-ec6e-4214-8234-5570e66e063c","version":4}
```

Comment:
Default validator keystores, just yoinked from the optimism repo. 

We need to generate these in cluster init container as i havent worked out a way to mount whole directories with relative paths